features/events/traps/blueprint-trapd-listener.xml

```diff
@@ -11,13 +11,39 @@
         http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
 ">
 
-    <cm:property-placeholder id="trapdProperties" persistent-id="org.opennms.netmgt.trapd" update-strategy="reload">
+    <cm:property-placeholder id="trapHandlerDefaultProperties" persistent-id="org.opennms.netmgt.trapd" update-strategy="none">
         <cm:default-properties>
             <cm:property name="trapd.listen.interface" value="127.0.0.1" />
             <cm:property name="trapd.listen.port" value="162" />
         </cm:default-properties>
     </cm:property-placeholder>
 
+    <!-- Camel Event for unmarshalling processor -->
+    <bean id="unmarshaller" class="org.opennms.core.camel.JaxbUtilsUnmarshalProcessor">
+        <argument value="org.opennms.netmgt.config.trapd.TrapdConfiguration" />
+    </bean>
+
+    <camelContext id="trapConnectionHandlerDefaultContext" xmlns="http://camel.apache.org/schema/blueprint">
+        <propertyPlaceholder id="minionproperties" location="blueprint:minionProperties" />
+        <route id="restClientToTrapdConfig">
+            <from uri="timer://restTrapConfigurationTimer?fixedRate=true&amp;period=60000"/>
+            <to uri="{{http-url}}/rest/config/trapd?authMethod=Basic&amp;authUsername={{username}}&amp;authPassword={{password}}" />
+            <bean ref="unmarshaller" />
+            <to uri="bean:trapdConfig?method=onUpdate" />
+            <bean ref="trapReceiver" method="setTrapdConfig" />
```

Comment:
As @j-white suggested in a previous PR, we should look into using the RestClient here instead of this Camel code since RestClient is easier to read and maintain.